pubspec.lock

```diff
@@ -643,7 +657,7 @@ packages:
       name: vector_math
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.8"
+    version: "2.1.0-nullsafety.2"
```

Comment:
out of curiosity what is the "nullsafety" bit? is this a stable version of the package?

Reply:
Just an update in the flutter sdk. It supports strong null safety now.